docs/demos/scripts/bootstrap.script

```diff
@@ -58,7 +58,7 @@ CLEAR
 ### STEP 1: download EasyBuild bootstrap script
 ###
 
-curl -O COPYPASTEhttps://raw.githubusercontent.com/hpcugent/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
+curl -O COPYPASTEhttps://raw.githubusercontent.com/easybuilders/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
```

Comment:
@boegel I think you missed a space between the protocol specification and `https`